client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-renderer.tsx

```diff
@@ -97,10 +98,20 @@ const PatternListRenderer = ( {
 	activeClassName,
 	composite,
 	onSelect,
+	isShowMorePatterns,
 }: PatternListRendererProps ) => {
+	const filterPriorityPatterns = ( pattern: Pattern ) => {
+		if ( isShowMorePatterns ) {
+			return pattern;
```

Comment:
It might be better to return true or false in the filter function. So we can simply return `isShowMorePatterns || isPriorityPattern( pattern )`

Reply:
I thought it was a map 😆